netty/src/main/java/io/grpc/transport/netty/Utils.java

```diff
@@ -137,6 +130,14 @@ public static Http2Headers convertClientHeaders(Metadata.Headers headers,
       http2Headers.path(new ByteString(headers.getPath().getBytes(UTF_8)));
     }
 
+    // Set the User-Agent header.
+    String userAgent = GRPC_USER_AGENT;
+    String applicationUserAgent = headers.get(USER_AGENT_KEY);
+    if (applicationUserAgent != null) {
+      userAgent = applicationUserAgent + " " + GRPC_USER_AGENT;
```

Comment:
In a discussion you weren't a part of, it was decided that the order would be:
language-UA + grpc-core-UA + application-UA

Since we don't have a separate library for language, we wouldn't have language-UA. But it would mean we should invert this concatenation order.